### PR TITLE
Add error handling in "query-paths" API

### PR DESCRIPTION
**Pull request by dongbohu**

### CODE REVIEW

File: dj_hetmech_app/views.py [hunk 1]
```diff
@@ -173,14 +173,27 @@ def get(self, request):
...
+                {'error': 'metapath parameter not found in URL'},
                 status=status.HTTP_400_BAD_REQUEST
             )
+        try:
+            Metapath.objects.get(pk=metapath)
+        except:
+            return Response(
+                {'error': 'metapath abbreviation not found in database'},
+                status=status.HTTP_404_NOT_FOUND
+            )
```
Comment (dhimmel) [inline dj_hetmech_app/views.py]: Can you undo this metapath check and revert to the TODO comment? The database does not contain all possible valid metapaths, so this is not the proper way to check.

Comment (dongbohu) [reply dj_hetmech_app/views.py]: Sure.

File: dj_hetmech_app/views.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from rest_framework.views import APIView
 from rest_framework.viewsets import ModelViewSet
 
-from .models import Node, PathCount
+from .models import Node, Metapath, PathCount
```
Comment (dhimmel) [inline dj_hetmech_app/views.py]: Can remove `Metapath` from import?

**Review by dhimmel — COMMENTED** [review]
One minor thing

File: dj_hetmech_app/views.py (hunk 2 above)
Comment (dongbohu) [reply dj_hetmech_app/views.py]: Done

**Pull request merged**